### ISSUE TITLE
paintcompiler is requiring an `fvar` table for static .ttf

### ISSUE THREAD

**Issue by vv-monsalve** [simoncozens/paintcompiler#6]
After trying to use a [paints.py](https://github.com/vv-monsalve/Kalnia-Color/blob/builder2/paintcompiler-test/paints.py) file with a [static .ttf font](https://github.com/vv-monsalve/Kalnia-Color/blob/builder2/sources/instance_ttf/KalniaColor-Regular.ttf) it fails with the following message: `AssertionError: Font needs an fvar table`

Complete traceback:

````
sources % paintcompiler instance_ttf/KalniaColor-Regular.ttf -p ../paintcompiler-test/paints.py -o instance_ttf/KalniaColorC-Regular.ttf
Adding paints...
Traceback (most recent call last):
  File "/Users/viv_monsalve/Documents/01_GOOGLE/GF_Projects/Kalnia/Kalnia-Color/venv/bin/paintcompiler", line 8, in <module>
    sys.exit(main())
  File "/Users/viv_monsalve/Documents/01_GOOGLE/GF_Projects/Kalnia/Kalnia-Color/venv/lib/python3.10/site-packages/paintcompiler/__init__.py", line 656, in main
    compile_paints(font, paints)
  File "/Users/viv_monsalve/Documents/01_GOOGLE/GF_Projects/Kalnia/Kalnia-Color/venv/lib/python3.10/site-packages/paintcompiler/__init__.py", line 572, in compile_paints
    builder = PythonBuilder(font)
  File "/Users/viv_monsalve/Documents/01_GOOGLE/GF_Projects/Kalnia/Kalnia-Color/venv/lib/python3.10/site-packages/paintcompiler/__init__.py", line 102, in __init__
    assert "fvar" in font, "Font needs an fvar table"
AssertionError: Font needs an fvar table
````

**Comment by vv-monsalve** [comment]
In the Builder2 branch (used in the above links), I've updated glyphsLib to `v6.4.1` which includes [#947](https://github.com/googlefonts/glyphsLib/pull/947) a fix for the `.color` unicode mapping for static fonts.

**Comment by simoncozens** [comment]
I've fixed this in 0.2.1.
